view/browse-page.html

```diff
@@ -82,179 +82,199 @@ <h2>Typing adjustment and how to make it easier</h2>
         <h1 class="articles-container-header">
           == Continue exploring below ==
         </h1>
-        <figure class="article-card">
-          <img
-            class="article-card-img"
-            src="../images/customizable1.jpg"
-            alt="compact keyboard"
-          />
-          <a href="article-page.html">
-            <h2>Why Mechanical Keyboards are Better for Gaming</h2>
-          </a>
-          <p>
-            Mechanical keyboards are great to use for gaming for a multitude of
-            reasons and are incredibly fun to play with.
-          </p>
-          <button class="article-card-expand-btn">
-            <a href="article-page.html">continue reading ...</a>
-          </button>
-        </figure>
-        <figure class="article-card">
-          <img
-            class="article-card-img"
-            src="../images/featureImage-Typing.jpeg"
-            alt="compact keyboard"
-          />
-          <a href="detail-page-light-sample1.html">
-            <h2>Lorem Ipsum Dolor sit amet</h2>
-          </a>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-            eiusmod tempor incididunt ut labore et dolore magna aliqua.
-          </p>
-          <button class="article-card-expand-btn">
+        <div class="article-card">
+          <figure>
+            <img
+              class="article-card-img"
+              src="../images/customizable1.jpg"
+              alt="compact keyboard"
+            />
+          </figure>
+          <div class="article-card-text">
+            <a href="article-page.html">
+              <h2>Why Mechanical Keyboards are Better for Gaming</h2>
+            </a>
+            <p>
+              Mechanical keyboards are great to use for gaming for a multitude
+              of reasons and are incredibly fun to play with.
+            </p>
+            <button class="article-card-expand-btn">
+              <a href="article-page.html">continue reading ...</a>
+            </button>
+          </div>
+        </div>
+        <div class="article-card">
+          <figure>
+            <img
+              class="article-card-img"
+              src="../images/featureImage-Typing.jpeg"
+              alt="compact keyboard"
+            />
+          </figure>
+          <div class="article-card-text">
             <a href="detail-page-light-sample1.html">
-              continue reading ...
+              <h2>Lorem Ipsum Dolor sit amet</h2>
```

Comment:
Then let's change this to `h3` since we are updating `Continue exploring below` to become an `h3`.